controllers/authController.js

```diff
@@ -0,0 +1,36 @@
+const authService = require("../services/authService");
+const fetch = require('node-fetch')
+const error = require("../middlewares/errorCreater");
+
+const logIn = async (req, res) => {
+    
+        const headers = req.headers["authorization"];
+        const accessToken = headers.split(" ")[1];
+
+        let nickname = ""
+        let kakaoId = ""
+        let profileImgUrl = ""
+
+       await fetch("https://kapi.kakao.com/v2/user/me", {
+            method: "GET",
+            headers: {
+              "Authorization": `Bearer ${accessToken}`,
+              "Content-type": "application/x-www-form-urlencoded;charset=utf-8",
+              },
+            })
+            .then(res => res.json())
+            .then(value => {
+              kakaoId= value.id
+              nickname = value.properties.nickname
+              profileImgUrl = value.properties.profile_image
+              
+            })
```

Comment:
```suggestion
       const response = await fetch("https://kapi.kakao.com/v2/user/me")
```
then으로 계속해서 콜백으로 코드를 늘려쓰면 가독성이 좋지 않습니다. 한번 다음과 같이 작성하여 활용해 보시겟어요?